README.md

```diff
@@ -406,6 +406,11 @@ set XDEBUG_CONFIG=idekey=PHPSTORM
 sudo -E -u www-data php -dxdebug.remote_host=192.168.21.1 occ
 ```
 
+### Logging
+
+The log level is set inside `data/additional.config.php` which is mounted as a read only volume in the container.
+To adjust the loglevel change it in the file and restart the container.
```

Comment:
Restart should be not needed, the next request will just read the new config value ;)

Reply:
Ahh... maybe i did not issue a request - but `occ config:system:get loglevel` still printed the old one.

Reply:
After changing the value back to two i get this:
```bash
www-data@c2de6db02cec:~/html$ ./occ config:system:get loglevel
0
www-data@c2de6db02cec:~/html$ grep loglevel config/additional.config.php 
	'loglevel' => 0,
```

Looks like the volume does not get updated even if the underlying file changes.

Reply:
Strange, works fine here. I'm wondering if that depends on how docker is setup to map mountpoints. On the other hand not updating would also cause quite some issues during development I'd assume.

Reply:
I figured out the problem actually occurs when using vim for updating the file as the inode changes with that, the mount to the container does no longer work

https://unix.stackexchange.com/questions/36467/why-inode-value-changes-when-we-edit-in-vi-editor

I refactored config handling a bit so that you can now put a custom user config in `data/shared/config.php` where the directoy is mounted in the container and then symlinked to the config directory which is a lot more stable.